openstack/identity/v3/tokens/requests.go

```diff
@@ -182,13 +182,13 @@ func Get(c *gophercloud.ServiceClient, token string) (r GetResult) {
 func Validate(c *gophercloud.ServiceClient, token string) (bool, error) {
 	resp, err := c.Request("HEAD", tokenURL(c), &gophercloud.RequestOpts{
 		MoreHeaders: subjectTokenHeaders(c, token),
-		OkCodes:     []int{204, 404},
+		OkCodes:     []int{200, 400, 401, 403, 404, 405, 413, 503},
```

Comment:
Is there a compelling reason not to go with @jtopjian 's suggestion of just using 200, 400, 401, 403, and 404?